Storage/Storage/Tools/StorageType+Extensions.swift

```diff
@@ -89,6 +89,13 @@ public extension StorageType {
         return firstObject(ofType: OrderStatsItem.self, matching: predicate)
     }
 
+    /// Retrieves the Stored OrderStats for V4 API.
+    ///
+    func loadOrderStatsV4(siteID: String, granularity: String) -> OrderStatsV4? {
+        let predicate = NSPredicate(format: "siteID = %ld AND ==[c] %@", granularity)
```

Comment:
shall the `siteID` be used as the first argument of the predicate format?

Reply:
Oh, wow, what was I thinking, that's wrong at so many levels... It should read:
```
let predicate = NSPredicate(format: "siteID = %ld AND granularity ==[c] %@", siteID, granularity)
```
Thanks!